sig-instrumentation/annual-report-2024.md

```diff
@@ -83,7 +83,7 @@ Operational tasks in [sig-governance.md]:
 - [x] Other contributing docs (e.g. in [devel] dir or contributor guide) reviewed for accuracy and updated if needed
 - [x] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
 - [x] SIG leaders (chairs, tech leads, and subproject leads) in [sigs.yaml] are accurate and active, and updated if needed
-- [ ] Meeting notes and recordings for 2024 are linked from [README.md] and updated/uploaded if needed
+- [x] Meeting notes and recordings for 2024 are linked from [README.md] and updated/uploaded if needed
```

Comment:
I find the template wording a bit confusing, as in, is “linked from README.md” meant to be, “linked here from README.md”, so we are supposed to put that link here as well? But we can come back to this later, just a nit.